libmscore/utils.cpp

```diff
@@ -760,9 +760,9 @@ Note* searchTieNote(Note* note)
                   QList<Chord*> gnb = c->graceNotesBefore();
                   if (!gnb.isEmpty()) {
                         Chord* gc = gnb[0];
-                        note2 = gc->findNote(note->pitch());
```

Comment:
Exactly the same construct here https://github.com/musescore/MuseScore/pull/2259/files#diff-4cbe68f0e843ff72e4c602279e7f35d2L738 but for grace note after. Is that a problem?